### ISSUE TITLE
elemental.dom.Node.addEventListener throws Uncaught ReferenceError

### ISSUE THREAD

**Issue by dankurka** [gwtproject/gwt#7515]
Originally reported on Google Code with ID 7518
```
elemental.dom.Node.addEventListener throws Uncaught ReferenceError, useCapture not defined
in Elemental, jsElementMixinBase.java.

Found in GWT Release (e.g. 2.4.0, 2.5.0 RC):

2.50.rc1, gwt-elemental.jar

Encountered on OS / Browser (e.g. WinXP, IE8-9, FF7):

Chrome, Windows, Vista, using Chrome dev tools and SourceMaps to debug code served
by CodeServer.

Detailed description (please be as specific as possible):

The generated javascript for the version of elemental.dom.Node.addEventListener(String
type, EventListener listener) that does NOT take a useCapture parameter, throws at
runtime on the browser.  Below is a text copy of the Chrome dev tools source (from
jsElementalMixinBase.java) with the exception message embedded.

You can see that the native code uses useCapture, but only type and listener are provided
by the java interface.

public native final EventRemover addEventListener(String type, EventListener listener)
/*-{
  var handler = @elemental.js.dom.JsElementalMixinBase::getHandlerFor(Lelemental/events/EventListener;)(listener);
  this.addEventListener(type, handler);
  return @elemental.js.dom.JsElementalMixinBase.Remover::create(Lelemental/events/EventTarget;Ljava/lang/String;Lcom/google/gwt/core/client/JavaScriptObject;Z)
[[Uncaught ReferenceError: useCapture is not defined]]
      (this, type, handler, useCapture);
}-*/;

Shortest code snippet which demonstrates issue (please indicate where
actual result differs from expected result):



Workaround if you have one:

Use the version of elemental.dom.Node.addEventListener() that takes a useCapture parameter,
so it is avaiable to the generated javascript JNI code.

Links to relevant GWT Developer Forum posts:



Link to patch posted at http://gwt-code-reviews.appspot.com:

```

Reported by `emurphy@conceptuamath.com` on 2012-07-14 02:56:17

**Comment by Artur-** [comment]
Why was the patch abandoned?

**Comment by Artur-** [comment]
Resubmitted as https://gwt-review.googlesource.com/#/c/14202/

**Comment by Artur-** [comment]
Fixed in 17601660c4804e9c35e8c39b2f9002b9617060f4
